.eslintrc

```diff
@@ -1,15 +1,12 @@
 {
-  "root": true,
-  "extends": "airbnb-base",
-  "plugins": [
-    "react"
-  ],
-  "env": {
-    "browser": true,
-    "es6": true,
-    "node": true,
-    "jasmine": true,
-    "jest": true
+  "plugins": ["patternfly-react"],
+  "extends": ["plugin:patternfly-react/recommended"],
+  "rules": {
+    "import/extensions": "off", # https://github.com/houndci/hound/issues/1419#issuecomment-328727830
```

Comment:
do we still care about hound issues (since we disabled it for JS?)

Reply:
Nice catch! Removed.